examples/kubernetes/dynamic-provisioning/filesystem/storageclass.yaml

```diff
@@ -4,22 +4,22 @@ metadata:
   name: fsx-sc
 provisioner: fsx.openzfs.csi.aws.com
 parameters:
-  volumeType: "filesystem" #REQUIRED
-  deploymentType: "SINGLE_AZ_1" #REQUIRED
-  throughputCapacity: "64" #REQUIRED
-  subnetIds: "subnet-0eabfaa81fb22bcaf" #REQUIRED
-  skipFinalBackupOnDeletion: "true" #REQUIRED, SET ON DELETION
-  optionsOnDeletion: "[DELETE_CHILD_VOLUMES_AND_SNAPSHOTS]" #SET ON DELETION
-  kmsKeyId: "12345678-90ab-cdef-ghij-klmnopqrstuv"
-  automaticBackupRetentionDays: "1"
-  copyTagsToBackups: "false"
-  copyTagsToVolumes: "false"
-  dailyAutomaticBackupStartTime: "00:00"
-  diskIopsConfiguration: "Iops=300,Mode=USER_PROVISIONED"
-  rootVolumeConfiguration: "CopyTagsToSnapshots=true,DataCompressionType=NONE,NfsExports=[{ClientConfigurations=[{Clients=*,Options=[rw,crossmnt]}]}],ReadOnly=false,RecordSizeKiB=128,UserAndGroupQuotas=[{Id=1,StorageCapacityQuotaGiB=10,Type=USER}]"
-  weeklyMaintenanceStartTime: "7:09:00"
-  securityGroupIds: "sg-068000ccf82dfba88"
-  tags: "Tag1=Value1,Tag2=Value2"
+  ResourceType: 'filesystem' #REQUIRED
```

Comment:
Hmm, I changed this because I didn't like how inconsistent it felt with the rest of the file. It'll be inconsistent regardless so I don't really feel strongly about this, I'll make the change